Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Surfaces/EnhancedSurface.azsli

```diff
@@ -24,20 +24,39 @@ class Surface
     float3 position;            //!< Position in world-space
     float3 normal;              //!< Normal in world-space
     float3 vertexNormal;        //!< Vertex normal in world-space
-    float3 albedo;              //!< Albedo color of the non-metallic material, will be multiplied against the diffuse lighting value
-    float3 specularF0;          //!< Fresnel f0 spectral value of the surface
+    float3 baseColor;           //!< Surface base color
+    float3 metallic;            //!< Surface metallic property
```

Comment:
Yea, the main reason to add this, IMO, is because these properties are needed later in the lighting pass, so this information would need to be exported if someone wanted to, for example, implement a deferred pipeline.